wpe/src/main/java/com/wpe/wpe/PageSettings.java

```diff
@@ -0,0 +1,89 @@
+package com.wpe.wpe;
+
+import android.annotation.SuppressLint;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PageSettings
+{
+    private static final int DefaultMajorVersion = 12;
+    private static final int DefaultMinorVersion = 0;
+    private static final int DefaultBugfixVersion = 0;
+    private static String OS_VERSION;
+
+    @SuppressLint("DefaultLocale")
```

Comment:
Instead of suppressing the warning, just specify the C locale when calling `String.format()` on line 34.
It will also prevent us from having strange numbers formatting if we encounter exotic default locales.